lib/src/di/di_clients/di_web_rtc_client.dart

```diff
@@ -15,8 +15,5 @@ Future<WebRtcClientChannel> _getWebRtcClient(
   final webRtcPeerConnection = WebRtcPeerConnection(webRtcDirectDataSource);
   await webRtcPeerConnection.createConnection();
 
-  return WebRtcClientChannel(
-    webRtcPeerConnection.peerConnection,
-    webRtcPeerConnection.dataChannel,
-  );
+  return WebRtcClientChannel(webRtcPeerConnection.peerConnection, webRtcPeerConnection.dataChannel, () => '');
```

Comment:
Why the empty function here? WebRtcClientChannel's returned here have no associated session ID?

Reply:
this is old Droids on roids code, naveed is actually deleting it here https://github.com/viamrobotics/viam-flutter-sdk/pull/68/files - so i just put this in as a placeholder, i should have called it out.

Reply:
Nice thanks.